sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodOverloadsShouldBeGrouped.vb

```diff
@@ -134,14 +134,37 @@ Namespace Tests.TestCases
 
     ' https://github.com/SonarSource/sonar-dotnet/issues/2776
     Public Class StaticMethodsTogether
```

Comment:
would it be useful to add `Func` tests as well?

Reply:
I'm sorry, I don't understand this point. Do you mean like this?
`public static int FunctionA() => 0;`

Reply:
yes, `Function` in addition to `Sub`